BTVNB4.css

```
@@ -0,0 +1,69 @@
+body {
+    background-color: #efefef;
+}
+div {
+    float: left;
+    padding: 30px;
+}
+.header {
+	width: 94%;
+	padding-left: 50%;
+	padding-right: 50%;
+
+
+}
+.center-1 {
+	width: 28%;
+	height: 150px;
+	float: left;
+	padding-left: 3%;
+	padding-right: 2%;
+	padding-top: 0%;
+
+}
+.center-2 {
+	width: 28%;
+	height: 150px;
+	float: left;
+	padding-left: 2%;
+	border-left:1px solid rgba(0, 0, 0, 0.418) ;
+	padding-right: 2%;
+	padding-top: 0%;
+}
+.center-3{
```

```suggestion
.center-3 {
```
chỗ thì e có cách, chỗ thì ko nhỉ